assets/js/modules/pagespeed-insights/components/DataSourceLearnMoreLinks.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Data Source Learn More Link components.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Link from '../../../components/link';
+import { sanitizeHTML } from '../../../util';
+
+function DataSourceLearnMoreLink( { href, html } ) {
+	return (
+		<Link
+			href={ href }
+			external
+			inherit
+			dangerouslySetInnerHTML={ sanitizeHTML(
+				html,
+				{
+					ALLOWED_TAGS: [ 'span' ],
+					ALLOWED_ATTR: [ 'class' ],
+				}
+			) }
+		/>
+	);
+}
+
+export function FieldDataLearnMoreLink() {
```

Comment:
I'm wary of exporting multiple components from the same file, as I know it can start to get a bit messy and harder-to-find components this way. But I get why it's happening here and it seems fine. Just something to keep an eye on I suppose, but this is fine 🙂

Reply:
I agree. It's not actually necessary anymore as the two will link to the same place so there will only be a single component 👍